tests/qmctl-test/files/qmctl_show_expected.txt

```diff
@@ -0,0 +1,104 @@
+[Install]
```

Comment:
I think this one could be deleted, this one not using it anymore
[qmctl_show.sh](https://github.com/containers/qm/pull/831/files#diff-96584ff33d9e524560fb602385d4c9310aac2e5b99c33f3ebccf9d82c8283720)